### ISSUE TITLE
MekHQ 47.7 won't load all units.

### ISSUE THREAD

**Issue by seekerofshadowlight** [MegaMek/megamek#2024]
### Environment
MekHq 47.7
Windows 10 ver 10.0 (x86)
Java version 1.8.0_251

### Description
The game will not load all the units. It loads my unit, then then the others controlled by the bots are not loaded at all or only some loaded, such as the opponent, but not my help, or one opponent group but not the second group
![MM11](https://user-images.githubusercontent.com/67355237/85499942-8d6fb000-b5b0-11ea-9309-730245d1d8ef.jpg)

sometimes exiting and reloading works, or it will get some units and other times I can do this more than ten times and no dice.

### Files

[Goldern knights30620205.zip](https://github.com/MegaMek/megamek/files/4823182/Goldern.knights30620205.zip)

**Comment by HammerGS** [comment]
NPE from Log
```
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
	at megamek.client.ui.swing.AccessibilityWindow$1.gamePlayerDisconnected(AccessibilityWindow.java:46)
	at megamek.common.event.GamePlayerDisconnectedEvent.fireEvent(GamePlayerDisconnectedEvent.java:39)
	at megamek.common.Game.processGameEvent(Game.java:3153)
	at megamek.client.Client.disconnected(Client.java:337)
	at megamek.client.Client$1$1.run(Client.java:203)
```

**Comment by sixlettervariables** [comment]
Potentially closed by #2025.
